src/lib/converter/converter.ts

```diff
@@ -338,6 +338,19 @@ export class Converter extends ChildableComponent<Application, ConverterComponen
             result.notSupportedIn = tag[1].split(/,\s?/);
         }
 
+        if (result && comment != null && comment.indexOf('@examples') != -1) {
+            var examplesTagRegex = /@(?:examples)\s*((?:'[^'\\]*(?:\\.[^'\\]*)*',\s*)*'[^'\\]*(?:\\.[^'\\]*)*')/;
```

Comment:
Use `const`

Reply:
Initially, I declared `examplesTagRegex` and `tag` as 'const', but for some reason, `tag` would always end up being null, which is why I ended up declaring them as 'var' (even 'let' wouldn't work). I may have been doing something wrong, though, because I see plenty of examples on the web where people are using const in similar instances.

I'll try again, we'll see :)